### PR TITLE
Add methods to timestamp class

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Requires #494 

No ColumnListTimestamp at the moment as heavydb doesn't support it. (cc @brenocfg)

### CODE REVIEW

**Review by tupui — APPROVED** [review]
This LGTM and I resolved the merge conflicts. I just relaunched failed jobs as it seems intermittent.

File: rbc/heavydb/timestamp.py [hunk 1]
```diff
@@ -1,17 +1,258 @@
 """HeavyDB Timestamp type that corresponds to HeavyDB type timestamp."""
```
Comment (brenocfg) [inline rbc/heavydb/timestamp.py:1]: Very minor but shouldn't this read `RBC Timestamp type that corresponds...`? As it is it's basically a tautology :p

**Review by brenocfg — APPROVED** [review]
Only suggested a small change in a comment but LGTM as well

File: rbc/heavydb/timestamp.py (hunk 1 above)
Comment (guilhermeleobas) [reply rbc/heavydb/timestamp.py]: Thanks!

**Pull request merged**